src/site/content/en/en.11tydata.js

```diff
@@ -16,14 +17,9 @@ const installable = require("../en/installable/installable.11tydata.js").path;
 // =============================================================================
 
 module.exports = function() {
-  const paths = [
-    fast,
-    accessible,
-    reliable,
-    secure,
-    discoverable,
-    installable,
-  ].filter(livePaths);
+  const paths = [fast, accessible, reliable, secure, discoverable, pwa].filter(
```

Comment:
Not super happy with the linter on this, but (shrug)

Reply:
hm yeah, prettier has some weird opinions... but I guess it's not worth fussing over